### ISSUE TITLE
Unable to stay logged in during local development

### ISSUE THREAD

**Issue by fgblomqvist** [resonatecoop/stream2own#155]
### Operating system
Linux

### Browser name and version
Firefox 96

### Expected behavior
I should be able to stay logged-in between page refreshes.

### Actual behavior
Anytime I refresh the page I am logged out again.

### Steps to reproduce behavior
1. Start up dev server
2. Login
3. Refresh the page (e.g. F5)
4. You're now logged out again

Not sure if this is only happening to me or not, or what could be the issue? Probably a cookie not being stored correctly?

**Comment by fgblomqvist** [comment]
Have you guys experienced this @peterklingelhofer @auggod?

**Comment by peterklingelhofer** [comment]
> Have you guys experienced this @peterklingelhofer @auggod?

Yes, this has always been the case on my setup. I've been guessing it's possibly because I don't have `nginx` set up for this project yet, figured it could be a cookies can only be set over HTTPS/SSL rule.

**Comment by fgblomqvist** [comment]
Hmm okay. Would be nice if it could be fixed/modified for local dev to avoid extra setup steps, but I imagine it can be hard if you're interacting with the prod API :thinking:
